pkgs/applications/misc/dbeaver/default.nix

```diff
@@ -23,16 +23,16 @@
   inherit maven; # use overridden maven version (see dbeaver's entry in all-packages.nix)
 }) rec {
   pname = "dbeaver";
-  version = "22.0.2"; # When updating also update mvnSha256
+  version = "22.0.5"; # When updating also update mvnSha256
 
   src = fetchFromGitHub {
     owner = "dbeaver";
     repo = "dbeaver";
     rev = version;
-    sha256 = "sha256-3tIxHw4734ggIUDjZO2EGIMbyPNP3yvy9QgnMLw+/fc=";
+    sha256 = "sha256-P+t1wtMTVKbfEXCFuz8zPNy3z4YysJRyOvTxiDFlp2c=";
   };
 
-  mvnSha256 = "os3eb+In8XreHwdZMacXafIVgOAeSSfCIkO5pwaO6MI=";
+  mvnSha256 = "sha256-veclFlzLhTU+nT360qxRNut+yEi2dfTBxdQASyRMqhI=";
```

Comment:
Did not test, maybe we need to remove the `sha256-` prefix:

https://github.com/NixOS/nixpkgs/blob/e8c877928079af54bf30782f453b59b0c3b188f2/pkgs/development/java-modules/maven-fod.nix#L10-L11

https://github.com/NixOS/nixpkgs/blob/e8c877928079af54bf30782f453b59b0c3b188f2/pkgs/development/java-modules/maven-fod.nix#L45

Reply:
Yes, build is failing. Fixed in https://github.com/RohanHart/nixpkgs/commit/04dd160e186a0edccaf06f32200922e0853ecfd7